tools/setup.py

```diff
@@ -113,8 +113,12 @@ def generate_gn_args(mode):
     if "DENO_BUILD_ARGS" in os.environ:
         out += os.environ["DENO_BUILD_ARGS"].split()
 
+    cacher = prebuilt.load_sccache()
+    if not os.path.exists(cacher):
```

Comment:
I think this branch is unnecessary -- if load_sccache() succeeds then this path *will* exist.

Reply:
It'd also be nice if we could somehow move sccache detection out of setup.py entirely. I think the goal should be to not rely on a generated args.gn file eventually.

Reply:
Yeah this branch is never taken, but I thought it would be good to leave it in if someone wanted to add some feature like `--no-prebuilt-sscache`

Reply:
I agree we should not rely on generated args.gn (other than is_official_release=true) - we can do that now by hard coding a default path to sscache now that we know it's going to be there.